_includes/sidebar.html

```diff
@@ -4,15 +4,17 @@
 <div class="panel panel-default">
   <div class="panel-heading">Latest release</div>
   <div class="panel-body">
-    <p>July 18, 2018<br /> 
-        4.3.1 - 
-        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.3.1.tar.gz']); void(0);" href="https://github.com/qutip/qutip/archive/v4.3.1.tar.gz">tar.gz</a>,
-        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.3.1.zip']); void(0);" href="https://github.com/qutip/qutip/archive/v4.3.1.zip">zip</a>, 
+    <p>July 18, 2018<br />
+        4.4.0 -
+        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.4.0.tar.gz']); void(0);" href="downloads/4.4.0/qutip-4.4.0.tar.gz">tar.gz</a>,
+        <!-- href="https://github.com/qutip/qutip/archive/v4.4.0.tar.gz" -->
+        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.4.0.zip']); void(0);" href="downloads/4.4.0/qutip-4.4.0.zip">zip</a>,
+        <!-- href="https://github.com/qutip/qutip/archive/v4.4.0.zip" -->
         <a href="docs/latest/index.html">docs</a>
-        (<a onclick="javascript:_gaq.push(['_trackEvent','download','qutip-doc','qutip-doc-4.3.pdf']); void(0);" href="downloads/4.3.1/qutip-doc-4.3.pdf">pdf</a>)
+        (<a onclick="javascript:_gaq.push(['_trackEvent','download','qutip-doc','qutip.pdf']); void(0);" href="downloads/4.4.0/qutip.pdf">pdf</a>)
```

Comment:
Is there some reason we did not call the file qutip-doc-4.4.pdf?

Reply:
Not really. That's the filename being build, why rename it?